src/store/builder/2019-10-10-shortcut-flow.json

```diff
@@ -1,15 +1,38 @@
 {
+  "specification_version": "1.0.0-rc4",
```

Comment:
Updated !
@jcbashdown , could you pls remind me what's the purpose of these json fixtures in flow-builder ?
For me, I just use them to manually test the importer feature. I don't know if the was an original purpose behind them.

Reply:
Same for me - I use them to test the importer and the various importer specific ways you can be prompted to resolve validation errors. It's possible that the one testing unmatched contact properties and groups is out of date but this will probably become relevant again when we reintroduce contact and group dropdowns in those blocks' uis.